.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,69 @@
+name: Bug Report
+description: Create a report to help us improve
+title: " "
+labels: ['type:Bug', 'status:Unconfirmed']
+body:
+  - type: textarea
+    id: "summary"
+    attributes:
+      label: Issue summary
+      description: "A brief summary of the issue"
+    validations:
+      required: true
+  - type: textarea
+    id: "steps"
+    attributes:
+      label: "Steps to reproduce"
+      value: |
+        1. Step 1
+        2. Step 2
+        3. ...
+    validations:
+      required: true
```

Comment:
Let's just keep the issue summary as required, and all other fields as optional. This reduces the friction while filing an issue, as the user won't need to fill something in all the field even for very simple issues.